plugins/inputs/logfile/tailersrc_test.go

```diff
@@ -208,11 +209,19 @@ func TestOffsetDoneCallBack(t *testing.T) {
 
 		if i == 15 { // Test after first truncate, saved offset should decrease
 			time.Sleep(1 * time.Second)
+			log.Println(statefile.Name())
 			b, err := ioutil.ReadFile(statefile.Name())
+			log.Println(b)
 			if err != nil {
 				t.Errorf("Failed to read state file: %v", err)
 			}
-			offset, err := strconv.Atoi(string(bytes.Split(b, []byte("\n"))[0]))
+			file_parts := bytes.Split(b, []byte("\n"))
+			log.Println("file_parts: ", file_parts)
+			file_string := string(file_parts[0])
+			log.Println("file_string: ", file_string)
+			offset, err := strconv.Atoi(file_string)
+			log.Println(offset)
+			log.Println(err)
```

Comment:
Do we need to log this? If `err` is not nil, then the line below will fail out the test.